CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Release Notes
 
+## Version 1.23.0 - 2021-09-03
+
+Features:
+* Add `--extra-disks` capability to kvm2 driver [#12351](https://github.com/kubernetes/minikube/pull/12351)
+* Support Rootless Docker [#12359](https://github.com/kubernetes/minikube/pull/12359)
+* Add support for tcsh in docker-env subcommand [#12332](https://github.com/kubernetes/minikube/pull/12332)
+* Support Ingress on MacOS, driver docker [#12089](https://github.com/kubernetes/minikube/pull/12089)
+* KVM2 driver for linux/aarch64 [#11768](https://github.com/kubernetes/minikube/pull/11768)
+* Build kicbase image for linux/s390x [#12079](https://github.com/kubernetes/minikube/pull/12079)
+* Add configurable port for minikube mount [#11979](https://github.com/kubernetes/minikube/pull/11979)
+* Add ability to create extra disks on hyperkit vms [#11483](https://github.com/kubernetes/minikube/pull/11483)
```

Comment:
move this to same under same bullet and use same wording as KVM one so it could say

```
- Add ability to create extra disks on hyperkit driver [#11483](https://github.com/kubernetes/minikube/pull/11483)
- Add ability to create extra disks on KVM driver

```

Reply:
done